packages/blazor-workspace/NimbleBlazor/NimbleBlazor.csproj

```diff
@@ -51,6 +51,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Include="CHANGELOG.json" />
```

Comment:
Just curious, but what is the purpose of including this in the project?

Reply:
Also curious, i.e. there is also a CHANGELOG.md file in this folder, what files do we need to pick and choose for exclusion?

Reply:
There may potentially be a node_modules in the folder as well next to the package.json. Would that cause issues in the future?

Reply:
I _thought_ that would prevent `CHANGELOG.json` from being copied to the output directory, which was ultimately causing a path conflict error when publishing the example. Turns out that didn't actually work, because despite VS showing that the file was set to "Do Not Copy", it sure as shit was getting copied. Eventually, I found [this](https://github.com/dotnet/sdk/issues/25587#issuecomment-1134731799), which after reading, I understood why we were already doing that for `package.json` and `package.lock.json`.

Reply:
Milan, apparently (see link in previous response) we only need to explicitly exclude `.json` files, `.config` files, and `wwwroot`.

Reply:
Can you add a comment to the file referencing why specifically the json files are being excluded.

Also the `**\*.json` matcher does make me worry about node_modules folders where the comment may have been missed.

Reply:
Can we just remove `**\*.json` instead of picking out individual json files?

Reply:
Yes, that's a good idea. I verified that there aren't (currently) any `.json` files other than the three previously listed, so that should be safe for now. I decided to do the same thing for `.config` files, too, since the `node_modules` concern would apply to them, too.